server/test/controllers/TaskSolverTest.scala

```diff
@@ -25,7 +26,7 @@ class TaskSolverTest extends PlaySpec with MockFactory with OneAppPerSuite {
         val description = "some description"
         val template = "some template"
         val replyTask = Task(year, scalaClass, timeuuid, "array", description, template, "ref", "test suite")
-        val taskSolver = new TaskSolver(mock[TestExecutor], dao, new MockMessageApi)
+        val taskSolver = new TaskSolver(mock[TestExecutor], dao, new MockMessageApi, new Provider[play.api.Application] { def get() = app})
```

Comment:
whent change this play.api.mvc.WebSocket#acceptWithActor
to play.api.libs.Streams.ActorFlow.actorRef
and than to Sink.actorRef and Source.actorRef